src/controllers/midi/midicontroller.cpp

```diff
@@ -497,19 +522,6 @@ double MidiController::computeValue(MidiOptions options, double _prevmidivalue,
     return _newmidivalue;
 }
 
-QString formatSysexMessage(const QString& controllerName, const QByteArray& data,
-                           mixxx::Duration timestamp) {
-    QString message = QString("%1: t:%2 %3 bytes: [")
-            .arg(controllerName).arg(timestamp.formatMillisWithUnit())
-            .arg(data.size());
-    for (int i = 0; i < data.size(); ++i) {
-        message += QString("%1%2").arg(
-            QString("%1").arg((unsigned char)(data.at(i)), 2, 16, QChar('0')).toUpper(),
-            QString("%1").arg((i < (data.size()-1)) ? ' ' : ']'));
-    }
-    return message;
-}
-
 void MidiController::receive(QByteArray data, mixxx::Duration timestamp) {
     controllerDebug(formatSysexMessage(getName(), data, timestamp));
```

Comment:
Uhh, Here is a real CPU burner. Please hide it behind  the ControllerDebug::enabled() flag